src/mailer/engine.py

```diff
@@ -13,6 +13,8 @@
 from django.core.mail import get_connection
 from django.core.mail.message import make_msgid
 from django.db import DatabaseError, NotSupportedError, OperationalError, transaction
+from django.utils.module_loading import import_string
```

Comment:
PS: Two now-unused imports above needs to be dropped [to make the CI happy](https://travis-ci.org/github/pinax/django-mailer/jobs/736107839#L201):
- `import smtplib`
- `from socket import error as socket_error`